### DISCUSSION TITLE
RGB goes dark on Commander Core & Elite Capellix XT

### DISCUSSION THREAD

**Discussion by matthlsile** [EvanMulawski/FanControl.CorsairLink#29] [Q&A]
Hi,

thank you for this great Plugin!

Yesterday i installed a Corsair 170i Elite Capellix XT which comes with a Commander Core. Pump and fans plus two case fans are attatched to the Commander Core. Sensors and fan/pump control work great.
But when starting Fancontrol the RGB goes dark on pump and fans. 

I'm not asking for rgb control. But it would be nice if RGB stayed on at least on the default setting.
Is this possible to achieve without a extra tool?

Kind regards

Matthl

**Comment by EvanMulawski** [comment]
Hi @matthlsile, there are two "modes" the Commander CORE and CORE XT can operate in:

**Hardware Mode** - this is the mode the device starts up in and will remain in this mode until a specific command is sent to the device

**Software Mode** - when a specific command is sent to the device, the device will be open to receiving LED and fan settings constantly as well as being able to report its current status

Each mode has its own settings/configuration for fans and lighting. For Fan Control to work, the device must be in Software Mode, as the CorsairLink plugin must constantly send fan settings and read fan RPM and temperature sensor values.

What you are experiencing is the device starting up in Hardware Mode and using that mode's lighting profile until Fan Control starts and the plugin sends the command to enter Software Mode, which will then cause the device to use the Software Mode lighting profile (which is not configured).

To solve this, you will need to use a separate application to configure RGB lighting (that isn't iCUE). SignalRGB version **2.2.29** and above is known to be compatible.

**Reply by matthlsile** [reply]
Hi @EvanMulawski, 

thank you for the explanation. 

Now that i understand the internals better, i see why it's not possible to implement my requested behaviour. ;-)

Matthl
